### PR TITLE
#134: Store regular schedule notes

### PR DESCRIPTION
**Pull request by awalkowiak**

### CODE REVIEW

File: src/main/java/org/benetech/servicenet/adapter/healthleads/HealthLeadsDataMapper.java [hunk 1]
```diff
@@ -94,6 +95,10 @@ public Service extractService(HealthleadsService service) {
         return srvc;
     }
 
+    public RegularSchedule extractRegularSchedule(HealthleadsLocation location) {
```
Comment (pmuchowski) [inline src/main/java/org/benetech/servicenet/adapter/healthleads/HealthLeadsDataMapper.java]: where is this method used?

File: src/main/java/org/benetech/servicenet/adapter/healthleads/persistence/PersistanceManager.java [hunk 2]
```diff
@@ -142,13 +143,17 @@ private Organization getOrganizationToPersist(HealthleadsOrganization healthlead
...
             String externalLocationId = healthleadsLocation.getId();
+            String schedule = healthleadsLocation.getSchedule();
 
             Location location = mapper.extractLocation(healthleadsLocation);
 
             location.setProviderName(PROVIDER_NAME);
             getPhysicalAddressesToPersist(externalLocationId).ifPresent(location::setPhysicalAddress);
             location.setPhones(getPhonesForLocationToPersist(externalLocationId));
             location.setLangs(getLanguagesForLocationToPersist(externalLocationId));
+            if (!schedule.isBlank()) {
```
Comment (pmuchowski) [inline src/main/java/org/benetech/servicenet/adapter/healthleads/persistence/PersistanceManager.java]: you can use StringUtils.isNotBlank to be null safe

File: src/main/java/org/benetech/servicenet/adapter/healthleads/HealthLeadsDataMapper.java [hunk 3]
```diff
@@ -156,4 +157,8 @@ public Service extractService(HealthleadsService service) {
 
     @Mapping(target = "id", ignore = true)
     protected abstract Phone toPhone(HealthleadsPhone phone);
+
+    @Mapping(target = "id", ignore = true)
+    @Mapping(target = "notes", source = "schedule")
+    protected abstract RegularSchedule toRegularSchedule(HealthleadsLocation location);
```
Comment (pmuchowski) [inline src/main/java/org/benetech/servicenet/adapter/healthleads/HealthLeadsDataMapper.java]: is this method used anywhere?

Comment (awalkowiak) [reply src/main/java/org/benetech/servicenet/adapter/healthleads/HealthLeadsDataMapper.java]: fixed

**Pull request merged**